cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md

```diff
@@ -12,7 +12,7 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
 
 > "I want to learn about the terms under which I can access and use the CMS Open Data, and publish results obtained from them."
 
-- Go to ["Data preservation and open access policy"](https://cms-docdb.cern.ch/cgi-bin/PublicDocDB/ShowDocument?docid=6032) (if you are a CMS member see also "papers by CMS members using public data").
+- Go to ["Data preservation and open access policy"](http://opendata-dev.web.cern.ch/record/411) (if you are a CMS member see also ["Rules for use of open access CMS data by individual members of CMS"](https://cms-docdb.cern.ch/cgi-bin/DocDB/ShowDocument?docid=12242)).
```

Comment:
You can exclude `http://opendata-dev.web.cern.ch` and just leave: `/record/411`, I think it should work at all places.

Reply:
OK, good, I will change them on next round. There is quite a lot to do still

Reply:
Done